src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -103,6 +109,10 @@ private void setFilingApiData(FilingApi filing, String requestId, String oversea
         boolean isUpdateFiling = setSubmissionData(data, overseasEntityId, logMap);
         setPaymentData(data, transaction, passThroughTokenHeader, logMap);
 
+        if (isUpdateFiling) {
```

Comment:
Ideally we should use the Dto to get the company number but ok for now. Maybe put entity_number in a constant meanwhile?